src/renderer/components/ChannelHome/ChannelHome.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <div
+      v-for="(shelf, index) in filteredShelves"
+      :key="index"
+    >
+      <details
+        :open="index == 0"
+      >
+        <summary
+          class="shelfTitle"
+        >
+          {{ shelf.title }}
+          <span
+            v-if="shelf.playlistId"
+            class="playAllSpan"
+          >
+            <router-link
+              class="playAllLink"
+              :to="{
+                path: `/playlist/${shelf.playlistId}`
+              }"
+            >
+              <FontAwesomeIcon
+                class="thumbnail"
+                :icon="['fas', 'play']"
+              />
+              {{ $t('Channel.Home.Play all') }}
```

Comment:
A bit misleading to say Play all since it goes to the playlist not start playing the first video
Youtube one does play 1st video

![image](https://github.com/user-attachments/assets/5f3c7e24-5032-4d69-9e55-5e2ab1ca168e)

Reply:
In YouTube, it's possible to view the playlist by clicking on the title of the shelf but in FreeTube that expands/collapses the shelf. I changed the link to a `View playlist` link since FreeTube users might want to convert a playlist to a FreeTube one/take full advantage of FreeTube playlists.

Reply:
Maybe capitalize the label like other buttons?
Also I think it might be better to use list icon like in playlist tab (to distinguish between online & local playlists) 

Label capitalized buttons
![image](https://github.com/user-attachments/assets/fcf1f764-ed26-475a-a24e-2d7a2b05f94d)

Preview:
![image](https://github.com/user-attachments/assets/f3006416-897e-4847-921b-1fe9ab3e87df)

Reply:
Done, I've switched to the list icon and capitalized the label